.github/workflows/release.yml

```diff
@@ -13,6 +13,21 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - build-os: linux
+            build-arch: amd64
+          - build-os: linux
+            build-arch: arm64
+          - build-os: linux
+            build-arch: s390x
+          - build-os: linux
+            build-arch: ppc64le
+          - build-os: linux
+            build-arch: riscv64
+          - build-os: linux
+            build-arch: static
```

Comment:
Do we actually need this?
I would assume that we don't, as for all the arches we're actually statically linking the binaries.

Another thing to be considered, is whether we need changes in the makefiles for building the content for different arches, as the changes done on #552.